runtime/vm/exceptionsupport.c

```diff
@@ -135,7 +135,7 @@ setCurrentExceptionNLSWithArgs(J9VMThread * vmThread, U_32 nlsModule, U_32 nlsID
 	nlsMsgFormat = omrnls_lookup_message(J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE,
 			nlsModule,
 			nlsID,
-			"");
+			NULL);
```

Comment:
We can't return NULL here as the `nlsMsgFormat` is used as the format string to the `j9str_vprintf` which doesn't check it for null.

Reply:
j9nls_lookup_message does not return NULL, it returns a failure string if the message cannot be found:
https://github.com/eclipse/omr/blob/57e8ed7a39d2f0570ff5218d3fc2758b0ef68e1c/port/common/j9nls.c#L401toL406,

Reply:
You're right.  If `parse_catalog` returns the `default_string`, null in this case, that code will return an error message.